src/manager.rs

```diff
@@ -56,6 +56,13 @@ impl<A: Actor> ActorManager<A> {
         addr
     }
 
+    /// Spawn the actor's main loop on the given runtime using the default spawner. This will allow
+    /// it to handle messages.
+    pub fn spawn_default<S: Spawner + Default>(self) -> Address<A> {
```

Comment:
This is not quite the same as with your original issue, right?

Reply:
Not really. Probably your extension trait suggestion `spawn_global` works better.

Reply:
Ok. I will try to get a basic implementation of it tomorrow.